build.rs

```diff
@@ -10,6 +10,8 @@ fn main() {
     if !is_docs_rs() && is_sidecar_ffi_enabled() {
         println!("cargo:rustc-cfg=f3sidecar");
         std::env::set_var("GOWORK", "off");
+        // See <https://github.com/golang/go/issues/58159>
+        std::env::set_var("GOFLAGS", "-tags=netgo");
```

Comment:
What's the pros and cons of using `netgo`? Could you add some comments on why `netgo` is enabled for `linux/amd64` and `linux/arm64` as well

Reply:
done, the are no cons to this really, especially because the sidecar does not heavily rely on name resolution.